src/component/addeditmembers/index.tsx

```diff
@@ -61,6 +61,7 @@ function AddEditMembers(props: AddEditMembersProps) {
 
   const { data: invitesData = [] } = useInvites({
     ventureId: currentVenture?.id,
+    timelineId: currentTimeline?.id,
```

Comment:
Part of the fix for https://github.com/venturemark/webclient/issues/418. Invites can now be part of a venture or part of a timeline so we have to specify the timeline (if it exists).